Public/Src/Engine/Processes/SandboxedProcessMac.cs

```diff
@@ -414,7 +414,7 @@ internal override JobObject.AccountingInformation GetJobAccountingInfo()
                         ReadTransferCount = Convert.ToUInt64(m_perfAggregator.DiskBytesRead.Total),
                         WriteTransferCount = Convert.ToUInt64(m_perfAggregator.DiskBytesWritten.Total)
                     }),
-                    PeakMemoryUsage = (ulong)m_perfAggregator.PeakMemoryBytes.Maximum,
+                    PeakMemoryUsage = Convert.ToUInt64(m_perfAggregator.PeakMemoryBytes.Maximum),

```

Comment:
>Convert.ToUInt64 [](start = 38, length = 16)

Convert.ToUInt64() still throws an OverflowException if PeakMemoryBytes is negative or too large. What does this change?

Reply:
Thats the point exactly, merely casting does not throw an exception. This way we can at least get a better idea of where bogus data is coming from.